README.md

```diff
@@ -2,7 +2,7 @@
 
 	GO111MODULE=on go get mvdan.cc/garble
 
-Obfuscate a Go build. Requires Go 1.13 or later.
+Obfuscate a Go build. Requires Go 1.14 or later.
```

Comment:
Might want to reword this to say that Go 1.14+ is supported, other versions may work but are not supported. Technically AFAIK garble doesn't require Go 1. 14 to build correctly

Reply:
Maybe - historically I've preferred to be more clear about the minimum requirements, because otherwise users might think this is just a suggestion. We don't test on older versions, so I don't want to get into what might or might not work.